docs/conf.py

```diff
@@ -82,6 +82,8 @@
     "sphinx.ext.graphviz",
     "sphinx_computron",
     "ref_fixer_hack",
+    "sphinxcontrib.mermaid",
+    "sphinxcontrib.contentui",
```

Comment:
If it's not maintained, then it should not be used. I am not aware of alternatives and I think the value of this feature is comparatively low, so I would not expend extra effort on implementing it. We have much more important things to do.